module/VuFind/src/VuFind/Db/Row/UserList.php

```diff
@@ -305,4 +308,118 @@ public function delete($user = false, $force = false)
         // Remove the list itself:
         return parent::delete();
     }
+
+    /**
+     * Get identifier (returns null for an uninitialized or non-persisted object).
+     *
+     * @return ?int
+     */
+    public function getId(): ?int
+    {
+        return $this->id ?? null;
```

Comment:
No need for the `?? null` here; if `$this->id` is already null, you don't need to coalesce it back to null again.